zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -786,8 +800,6 @@ public void updateStatus(char initial) {
                 if (version.getStatus() != EntityStatus.OBSOLETE) {
                     version.setStatus(EntityStatus.OBSOLETE);
                     entityManager.merge(version);
-                    projectIterationUpdateEvent.fire(
-                            new ProjectIterationUpdate(version));
```

Comment:
It has moved to SlugEntityUpdateListener, the hibernate event interceptor. Here it will only fire event when you update project in JSF but not from REST. I think it will be more robust to put it in the new place.